core/src/lib.rs

```diff
@@ -170,25 +230,33 @@ pub struct InputState {
     pub play: bool,
 }
 
+// A game from the perspective of a specific player
+pub struct GameAsPlayer {
```

Comment:
Are there any other users of this struct aside from the client?

Reply:
There's a test use at the bottom of this file.